plugins/plugin-site/src/utils/algolia-queries.js

```diff
@@ -38,16 +41,12 @@ function pluginQueries() {
             {
                 type: 'synonym',
                 synonyms: ['perforce', 'p4'],
-                objectID: 'syn-1617250859718-18'
-            }
+                objectID: 'syn-1617250859718-18',
+            },
         ],
         settings: {
             paginationLimitedTo: 2000, // they recommend 1000, to keep speed up and prevent people from scraping, but both are fine to us
-            optionalWords: [
-                'jenkins',
-                'plugin',
-                'plugins'
-            ],
+            optionalWords: ['jenkins', 'plugin', 'plugins'],
```

Comment:
for the future, unrelated changes make reviewers nervious/harder to review. 
So i recommend using `git add -p` to only commit the lines you changed.

Reply:
Got it